### PR TITLE
Video List

### PR DESCRIPTION
**Pull request by JacobJaffe**
Adds the video list, which refactors `NavCard` -> `Card`

Needs copy for some language strings still, pending design
image
image

Next steps: 
- wire up full movie playing

### CODE REVIEW

File: data/videos.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const videos = [
+  {
+    title: 'Know Your Rights (general tips)',
```
Comment (kiwansim) [inline data/videos.js:3]: Could you toss the video titles into en.json?

File: js/components/Card.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  style: ViewPropTypes.style,
+};
+
+const styles = StyleSheet.create({
+  card: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    padding: 12,
+    backgroundColor: '#FFFFFF',
```
Comment (kiwansim) [inline js/components/Card.js:27]: import from color style file!

File: js/components/Card.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    borderRadius: 3,
+    borderWidth: 2,
+    borderColor: '#eeeeee',
```
Comment (kiwansim) [inline js/components/Card.js:30]: same here re: color style file

File: lang/en.json [hunk 4]
```diff
@@ -49,6 +49,9 @@
     "header_work": "At Work",
     "header_scenarios": "Scenarios",
 
+    "header_videos": "Videos",
```
Comment (kiwansim) [inline lang/en.json:52]: Could you add the new keys to the Klingon file as well?  Helpful to have the two files have the same json structure even if it's mock content

**Comment by JacobJaffe** [comment]
@kiwansim this is updated btw!

**Pull request merged**